gobuffalo/azuredeploy.json

```diff
@@ -0,0 +1,196 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "imageName" : {
+        "type": "String",
+        "defaultValue": "appsvc/sample-hello-world:latest",
+        "metadata": {
+          "description": "The image which defines the application to be hosted in Azure."
+        }
+      },
+      "name": {
+        "type": "String",
+        "defaultValue": "[concat('site', uniqueString(resourceGroup().id, deployment().name))]",
+        "metadata": {
+          "description": "A unique name to identify the site, and its relate assets once it is created."
+        }
+      },
+      "database": {
+        "type": "String",
+        "defaultValue": "none",
+        "allowedValues": [
+            "none",
+            "postgresql",
+            "mysql"
+        ],
+        "metadata": {
+          "description": "The flavor of database that should be deployed alongside this application."
+        }
+      },
+      "databaseName": {
+        "type": "String",
+        "defaultValue": "buffalo_development",
+        "metadata": {
+          "description": "The name that will identify the database which is created, should one be created."
+        }
+      },
+      "databaseAdministratorLogin": {
+          "type": "String",
+          "defaultValue": "[concat('admin', parameters('name'))]",
+          "metadata": {
+            "description": "The user handle for the administrator of the database to be created."
+          }
+      },
+      "databaseAdministratorLoginPassword": {
+        "type": "SecureString",
+        "defaultValue": "",
+        "metadata": {
+          "description": "The password for the administrator of the database to be created."
+        }
+      }
+    },
+    "variables": {
+      "hostingPlanName": "[concat('hostingPlan-', parameters('name'))]",
+      "postgresqlName": "[concat(parameters('name'), '-postgresql')]",
+      "mysqlName": "[concat(parameters('name'), '-mysql')]",
+      "postgresqlConnection": "[concat('postgres://', parameters('databaseAdministratorLogin'), '@', variables('postgresqlName'),':', parameters('databaseAdministratorLoginPassword'), '@', variables('postgresqlname'), '.postgres.database.azure.com:5432/', parameters('databaseName'))]",
+      "mysqlConnection": "[concat('mysql://', parameters('databaseAdministratorLogin'), '@', variables('mysqlName'), ':', parameters('databaseAdministratorLoginPassword'), '@tcp(', variables('mysqlName'), '.mysql.database.azure.com:3306)/', parameters('databaseName'), '?allowNativePasswords=true')]"
+    },
+    "resources": [
+      {
+        "type": "Microsoft.Web/sites",
+        "name": "[parameters('name')]",
+        "apiVersion": "2016-03-01",

```

Comment:
update apiVersion

Reply:
I used only API Versions that were provided after I created these sites through the portal. I'm hesitant to deviate from that behavior. Do you have particular knowledge of the API Version I should be choosing?